### ISSUE TITLE
[ApiView] Swagger definition missing a property

### ISSUE THREAD

**Issue by tg-msft** [Azure/azure-sdk-tools#4781]
Take a look at https://apiview.dev/Assemblies/Review/7c11c4c760344def8e4ce79217d2e805/fb63c85ea69e493b8d8e11b2b0a965c0#-azureopenai-2022-12-01.swagger-Definitions-FileStatistics which renders as:
![image](https://user-images.githubusercontent.com/1179329/203418674-57883a45-dbc7-4079-bc86-1d299dbe484a.png)
Even though the definition is:
```json
    "FileStatistics": {
      "title": "FileStatistics",
      "description": "A file is a document usable for training and validation. It can also be a service generated document with result details.",
      "type": "object",
      "properties": {
        "tokens": {
          "format": "int32",
          "description": "The number of tokens used in prompts and completions for files of kind \"fine-tune\" once validation of file content is complete.",
          "type": "integer",
          "readOnly": true
        },
        "examples": {
          "format": "int32",
          "description": "The number of prompts with completions for files of kind \"fine-tune\" once validation of file content is complete.",
          "type": "integer",
          "readOnly": true
        }
      }
    },
```

**Comment by ruowan** [comment]
Sure. I will take a look.

**Comment by ruowan** [comment]
This has already been fix in swaggerApiParser 1.0.3 version. Maybe after the next apiView release, you could get the correct result.
